lib/javascripts/base_app.js

```diff
@@ -42,6 +43,11 @@ function registerHelpers(app) {
       return e.message;
     }
   });
+
+  Handlebars.registerHelper('spinner', function(key) {
+    if (_.isObject(key)) { key = "dotted"; }
```

Comment:
Not sure, it seems key is an object if you don't pass a type option to the helper. "dotted" is the only supported option in any case, so I'll just remove it.